WordPress/Classes/Utility/Keychain.swift

```diff
@@ -0,0 +1,45 @@
+@objcMembers
+class Keychain {
```

Comment:
Fair point, I can rename it but naming is difficult 😆. What do you think about `KeychainUtils`?

Reply:
That's why I wasn't able to come up with a suggestion too :D KeychainUtils makes sense I think as it is a replacement for the SFHF one. The parameter might look odd indeed though I'd prefer pushing confusion one layer down rather than having it on the class name.